mockative/mockative
# Release of Mockative version 1.2.0

### Changes
- Completely rewritten code generation to use [KotlinPoet](https://square.github.io/kotlinpoet/)
- Update Kotlin and KSP versions to `1.6.20`
- Support incremental processing by splitting generated code into separate files and reducing file dependencies
- Add function to test wildcard generics to address #29
- Add support for extension receivers, fixing #30
- Add an `expect/actual` interface to address #31
